src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -5,26 +5,26 @@
 import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameAndTagContainsKeywordsPredicate;
+import seedu.address.model.person.SearchPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Finds and lists all persons in address book whose name, tags or company name contains any of the argument keywords.
  * Keyword matching is case-insensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Finds all persons whose names, tags or company names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: TYPE/KEYWORD TYPE/KEYWORD ... \n"
-            + "Where TYPE can be n/ or t/ to specify name or tag respectively.\n"
-            + "Example: " + COMMAND_WORD + " n/alice n/bob t/friends t/owesMoney \n"
-            + "Please Refer to User Guide for more details.";
+            + "Parameters: n/KEYWORD [MORE_KEYWORDS]... t/KEYWORD [MORE_KEYWORDS]...\n"
```

Comment:
Maybe we should add c/KEYWORD [MORE_KEYWORDS] as well

Reply:
Fixed

Reply:
fixed